scripts/west_commands/runners/jlink.py

```diff
@@ -77,6 +81,8 @@ def do_add_parser(cls, parser):
                             help='if given, GDB uses -tui')
         parser.add_argument('--gdbserver', default='JLinkGDBServer',
                             help='GDB server, default is JLinkGDBServer')
+        parser.add_argument('--gdb-host', default="",
```

Comment:
Please remove the `default` kwarg here. The default of `None` should be fine.

Reply:
Actually, this is an issue. If `None` is passed to `format` it will print out the literal `None`. So the default empty string is required.